src/concept/thing/Attribute.jl

```diff
@@ -27,3 +27,16 @@ function Attribute(t::Proto.Thing)
         throw(TypeDBClientException(CONCEPT_BAD_VALUE_TYPE))
     end
 end
+
+function get_owners(transaction::AbstractCoreTransaction,
+                    attribute::AbstractAttribute,
+                    thing_type::Optional{AbstractThingType} = nothing)
```

Comment:
BlueStyle indentation. See https://github.com/invenia/BlueStyle/blob/master/README.md#method-definitions

Similar comment for other places.